### PR TITLE
🔀🔖 4.7.0

### PR DESCRIPTION
**Pull request by jxn-30**
**Releasenotes for this release**
* ✨ turn closed Beta into open Beta => branch chooser in settings
* 🐛 [heatmap] fix multiple settings windows
* 🐛 [hotkeys] fix multiple help windows (F1)
* ✨ [em] mapCSSFilter
* ✨ [em/arrSearch] optional actions on Enter
* ✨ [ecw/tt,ecw/ai] allow type without custom categories
* ✨ [extendedBuilding] startPatrolsShortcut
* ✨ [hotkeys] vehicles
* ✨ [redesign/vehicles] support starrableMissions

---
**Releasenotes for translation**
```md
* ✨ turn closed Beta into open Beta => branch chooser in settings
* 🐛 [$m('heatmap')] fix multiple settings windows
* 🐛 [$m('hotkeys')] fix multiple help windows (F1)
* ✨ [$m('extendedMap')] $s(!'extendedMap.mapCSSFilter')
* ✨ [$s('extendedCallWindow.arrSearch')] optional actions on Enter
* ✨ [$s('extendedCallWindow.tailoredTabs'), $s('extendedCallWindow.alarmIcons')] allow type without custom categories
* ✨ [$m('extendedBuilding')] startPatrolsShortcut
* ✨ [$m('hotkeys')] vehicles
* ✨ [$m('redesign')] support starrableMissions
```

### CODE REVIEW

**Review by kdev — CHANGES_REQUESTED** [review]

**Review by kdev — APPROVED** [review]

File: docs/de_DE/settings.md [hunk 1]
```diff
@@ -5,13 +5,17 @@ lang: de_DE
 
 # Einstellungen :gear:
 
-In den Einstellungen werden die Einstellungen aller Module zentral verwaltet. Es können nur die Einstellungen aktiver Module verändert werden.
+In den Einstellungen werden die Einstellungen aller Module zentral verwaltet. Es können nur die Einstellungen aktiver
```
Comment (Cr4zyc4k3) [inline docs/de_DE/settings.md:8]: "fast aller Module"
Eg. Heatmap has external settings

File: src/components/setting/slider.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            <template v-else>123</template>
+        </button>
+        <label>
+            <input
+                :name="name"
+                :placeholder="placeholder"
+                :min="min"
+                :max="max"
+                :step="
+                    rangeMode ? step : typeof step === 'number' ? step : 0.001
```
Comment (Cr4zyc4k3) [inline src/components/setting/slider.vue:23]: Ninja coding par excellence ;)

File: src/i18n/en_GB/buildings.ts [hunk 3]
```diff
@@ -121,7 +121,7 @@ export default {
                 unlocksVehicleTypes: [33, 34],
                 givesParkingLots: 2,
                 isVehicleExtension: true,
-                parkingLotReservations: [33, 34],
+                parkingLotReservations: [[33], [34]],
```
Comment (Cr4zyc4k3) [inline src/i18n/en_GB/buildings.ts:124]: Nested array is correct?

File: src/i18n/en_GB/buildings.ts [hunk 4]
```diff
@@ -593,10 +593,10 @@ export default {
                 credits: 150_000,
                 coins: 20,
                 duration: '5 Days',
-                unlocksVehicleTypes: '33, 34',
-                givesParkingLots: '2',
+                unlocksVehicleTypes: [33, 34],
+                givesParkingLots: 2,
                 isVehicleExtension: true,
-                parkingLotReservations: '33, 34',
+                parkingLotReservations: [[33], [34]],
```
Comment (Cr4zyc4k3) [inline src/i18n/en_GB/buildings.ts:599]: Same here

File: src/i18n/fr_FR/buildings.ts [hunk 5]
```diff
@@ -635,12 +636,11 @@ export default {
         ],
         maxBuildings: 'Aucune limite',
         maxLevel: 20,
-        special:
-            'Le prix de vos postes augmente lorsque vous en possédez 25. Ainsi, le rythme de progression est constant une fois que vous possédez un grand flux de revenus à ces niveaux de jeu. La formule actuelle pour calculer le prix des postes est la suivante : <code>(50 000+100 000*LOG<sub>2</sub>(Number of existing fire stations − 22))</code>.',
+        special: '',
```
Comment (Cr4zyc4k3) [inline src/i18n/fr_FR/buildings.ts:639]: Specialisation removed?

File: src/modules/extendedBuilding/assets/startPatrolsShortcut.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    patrolBtn.textContent = $m('startPatrols.title').toString();
+
+    const caret = document.createElement('span');
+    caret.classList.add('caret');
+    patrolBtn.append(caret);
+
+    const patrolDropdown = document.createElement('ul');
+    patrolDropdown.classList.add('dropdown-menu', 'pull-right');
+
+    [3600, 10_800, 21_600].forEach(duration => {
```
Comment (Cr4zyc4k3) [inline src/modules/extendedBuilding/assets/startPatrolsShortcut.ts:19]: _ as seperator?

File: src/modules/extendedCallWindow/assets/arrSearch.ts [hunk 7]
```diff
@@ -109,6 +113,23 @@ export default (
...
+            searchField.addEventListener('keyup', e => {
+                if (e.key !== 'Enter') return;
+                if (selectOnEnter) {
+                    document
+                        .querySelector<HTMLAnchorElement>(
+                            `.aao_searchable:not(${hideSelector})`
+                        )
+                        ?.click();
+                }
+                if (clearOnEnter) {
```
Comment (Cr4zyc4k3) [inline src/modules/extendedCallWindow/assets/arrSearch.ts:126]: Should be an "else if"

**Review by Cr4zyc4k3 — APPROVED** [review]

File: src/i18n/en_GB/buildings.ts (hunk 3 above)
Comment (jxn-30) [reply src/i18n/en_GB/buildings.ts:124]: yes. Is the intended way.

File: src/modules/extendedBuilding/assets/startPatrolsShortcut.ts (hunk 6 above)
Comment (jxn-30) [reply src/modules/extendedBuilding/assets/startPatrolsShortcut.ts:19]: yes. enforced by ESLint. Makes larger numbers better readable.

File: src/modules/extendedCallWindow/assets/arrSearch.ts (hunk 7 above)
Comment (jxn-30) [reply src/modules/extendedCallWindow/assets/arrSearch.ts:126]: they are not exclusive so this is fine.

**Pull request merged**